### ISSUE TITLE
Add possibility to skip changelog for pre releases

### ISSUE THREAD

**Issue by Deadvi5** [versionize/versionize#113]
In order to avoid duplications in the changelog, it would be nice to have an options like `--skip-changelog` that allow to skip changelog writing with pre-releases.

To summarize with an example:

```
versionize --pre-release alpha
versionize --pre-release alpha
versionize --aggregate-pre-releases
```

Will actually produce 

v1.0.0-alpha.0
- featA
v1.0.0-alpha.1
- featB
v1.0.0
- featA
- featB

Request scope is

```
versionize --pre-release alpha --skip-changelog
versionize --pre-release alpha --skip-changelog
versionize --aggregate-pre-releases
```

Will produce

v1.0.0
- featA
- featB

A more optimal alternatives is an option that delete alpha.0 and alpha.1 from the changelog when a stable is released. In this way we can see changelog of --pre-release until a stable is released.

The flow will be

```
versionize --pre-release alpha 
versionize --pre-release alpha
```

v1.0.0-alpha.0
- featA
v1.0.0-alpha.1
- featB

```
versionize --aggregate-pre-releases --skip-prerelease-changelog
```

v1.0.0
- featA
- featB

**Comment by cabauman** [comment]
Hey, thanks for sharing the idea. We'll look into it.

**Comment by cabauman** [comment]
resolved by #127 (added `--skip-changelog` option)
